multiversx_sdk_cli/cli_shared.py

```diff
@@ -360,6 +361,58 @@ def prepare_guardian_account(args: Any):
     return account
 
 
+def load_guardian_account(args: Any) -> Union[Account, None]:
+    if args.guardian_pem:
+        return Account(pem_file=args.guardian_pem, pem_index=args.guardian_pem_index)
```

Comment:
Oh, now I see, this is the `Account` defined here in mxpy, not in sdk-py.

Reply:
Maybe at some point we can use the one defined in sdk-py?

Reply:
Indeed, will keep in mind for a future refactoring.